packages/xstate-graph/src/graph.ts

```diff
@@ -516,11 +544,10 @@ export function traverseSimplePlans<TState, TEvent extends EventObject>(
   > = {};
 
   function util(
-    fromState: TState,
-    toStateSerial: SerializedState,
-    event: TEvent | null
```

Comment:
We no longer need the `event` here because we have the `fromStateSerial` directly. This is a bit of an optimization too: we no longer need to serialize the state because it's already serialized from the adjacency map.